code/core/src/components/components/Card/Card.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { type ComponentProps, forwardRef } from 'react';
+
+import type { CSSObject, color } from 'storybook/theming';
+import { keyframes, styled } from 'storybook/theming';
+
+const fadeInOut = keyframes({
+  '0%': { opacity: 0 },
+  '5%': { opacity: 1 },
+  '25%': { opacity: 1 },
+  '30%': { opacity: 0 },
+});
+
+const spin = keyframes({
+  '0%': { transform: 'rotate(0deg)' },
+  '10%': { transform: 'rotate(10deg)' },
+  '40%': { transform: 'rotate(170deg)' },
+  '50%': { transform: 'rotate(180deg)' },
+  '60%': { transform: 'rotate(190deg)' },
+  '90%': { transform: 'rotate(350deg)' },
+  '100%': { transform: 'rotate(360deg)' },
+});
+
+const slide = keyframes({
+  to: {
+    backgroundPositionX: '36%',
+  },
+});
+
+interface CardProps extends ComponentProps<typeof Content> {
```

Comment:
This is a new public facing component, we should carefully consider it's interface, as this will have to follow semver.

Is outlineStyles a prop we want to have? It seem conflicting with the other props, also about styling the outline.